resources/assets/coffee/react/mp-history/game.coffee

```diff
@@ -50,7 +50,7 @@ class MPHistory.Game extends React.Component
             lookupUser: @props.lookupUser
             key: m.slot
 
-      if showTeams
+      if showTeams && @props.event.game.data.end_time
```

Comment:
this is the other optional missing `?`, btw. (though realistically speaking it should never be 0 - hence optional)